PR: Add videos on profiling and code analysis to "Working with Spyder" section (#175)
spyder-ide/spyder-docs: doc/working-with-spyder.rst

* Quickly navigate within and between files with the Outline pane
* Search for text or regular expressions across your entire project with the Find pane
* Discover and explore structured documentation in the Online Help pane

===========================
Improving your code quality
===========================

Learn how to improve the quality of your programs using code analysis.

.. youtube:: JnljnAjdO_w
            :height: 360
            :width: 640
            :align: left

* Open and use Code Analysis to evaluate the quality and style of Python files

> * Run analysis on a file in the Editor, on your computer or at a path

Reviewer: ```suggestion
* Run analysis on a file in the Editor or in your computer
```

Author: @CAM-Gerlach was the one who wrote it so I think "on" is right. I'll let him confirm this and adjust your suggestion accordingly.

Reviewer: Ok, it's just that the current text is a bit complicated to understand.

Author: Yeah, I agree with removing the "or at a path" part.

Reviewer 2: ```suggestion
* Run analysis on a file in the Editor or anywhere on your computer
```

@juanis2112 is right, "on" is the correct word in this context. I also agree with simplifying it; I've added "anywhere" to help balance the phrasing

Reviewer: Agreed. I was thinking to suggest something along those lines, but I couldn't come up with a sentence that sounded good to me.

* Determine what an error, warning or message means and how to fix it
* Turn off messages on a line, in a file or globally

====================
Optimizing your code
====================

> Learn how to optimize your code using the profiler.

Reviewer: ```suggestion
Learn how to optimize your code using the Profiler.
```

Author: Agreed

Reviewer 2: Also agreed

.. youtube:: 4hcXa8aGu_c
            :height: 360
            :width: 640
            :align: left

> * Open and utilize the Profiler to determine the bottlenecks in your programs
> * Run profiling on a file in the Editor, on your computer or at a path

Reviewer: ```suggestion
* Use the Profiler to determine bottlenecks in your programs
* Run profiling on a file in the Editor or in your computer
```

Author: Same as before, I'll wait for @CAM-Gerlach to see if "on" is right.

Reviewer 2: ```suggestion
* Use the Profiler to find bottlenecks in your programs
* Run profiling on a file in the Editor or elsewhere on your machine
```

Yep, same as before @juanis2112 is correct, and I agree with simplifying it. I changed "determine" to "find" since it was a more appropriate word with the revised phrasing (determine is a more complicated word and needs "the"), and made the wording of the second phrase less duplicative of the previous video (my fault originally).
